meta-iot2000-example/recipes-example/iot2000setup/files/iot2000setup.py

```diff
@@ -489,7 +488,11 @@ def configureNetworkInterfaces():
 	
 
 def expandFileSystem():
```

Comment:
Can't this method be removed as it's not selectable from the menu any longer as per above changes?